### DISCUSSION TITLE
Offer hint for `No summary of scanned files reported by`

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#52022] [Code Security]
https://github.com/jsoref/broken-workflows/security/code-scanning/tools/check-spelling/status

check-spelling
Scanned files
No summary of scanned files reported by check-spelling
Setup types
Check Spelling

Actions workflow · Last scan 24 minutes ago

![Screenshot_20230405-022621.png](https://user-images.githubusercontent.com/2119212/229998438-f52ed3ec-a8c9-46d3-858a-25ca02599274.png)

Neither Google nor GitHub seem to recognize `No summary of scanned files reported by`.

It sounds interesting, and I'd be happy to add support for it if there was a pointer provided.

**Comment by Athari** [comment]
Amusingly, this is still the only place on the Internet where the "No summary of scanned files reported by" message is mentioned.
